src/main/java/com/exadel/discountwebapp/cloud/CloudController.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discountwebapp.cloud;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController()
+@RequestMapping("/api/cloud")
```

Comment:
let's rename to something like "/api/image-upload"

Reply:
done